src/core/server/integration_tests/saved_objects/migrations/check_registered_types.test.ts

```diff
@@ -124,7 +124,7 @@ describe('checking migration metadata changes on all registered SO types', () =>
         "osquery-saved-query": "7b213b4b7a3e59350e99c50e8df9948662ed493a",
         "query": "4640ef356321500a678869f24117b7091a911cb6",
         "sample-data-telemetry": "8b10336d9efae6f3d5593c4cc89fb4abcdf84e04",
-        "search": "d26771bcf7cd271162aab3a610b75249631ef6b1",
+        "search": "b212646cedccdfb0f8504426f72d1e087c4bab7c",
```

Comment:
Curious how this change was introduced?

Reply:
These are hashes of saved object mappings to alert the core team when the mappings are updated. The hash got updated once when we added `breakdownField` to the PR, but it got updated again when I merged main recently since we also added `usesAdHocDataView`. Basically I just needed to update it again to match the latest hash.